### DISCUSSION TITLE
Changing layout for active display

### DISCUSSION THREAD

**Discussion by gugahoi** [asmvik/yabai#1462] [Q&A] [answered]
I was wondering if there is a way to change the layout only for the current active monitor. For context I work with 3 displays and frequently change the layout of all of them via:

```
shift + alt - f : yabai -m config layout stack
shift + alt - d : yabai -m config layout bsp
```

I would like to have the ability to change the layout of only a single display that is currently active but am unsure how to go about that. 
Any help is appreciated.

**Comment by dmitrym0** [comment, marked as answer]
You can specify per space config:

```
yabai -m space --layout stack
```
